gradle-plugin/doc/src/main/asciidoc/inc/scenarios/_quarkus.adoc

```diff
@@ -0,0 +1,74 @@
+[[quarkus-scenario]]
+== Quarkus
+JKube provides ready to use generators. A JKube generator is composed of default configurations to build container images, generate k8s resources and deploy them to kubernetes clusters. There are predifined generators for each type of Java project. For instance, we have a generator for Quarkus projects. It helps JKube to identify that a project is a Quarkus one and provides the right configuration to be used.
```

Comment:
> A JKube generator is composed of default configurations to build container images, generate k8s resources and deploy them to kubernetes clusters.


I think maybe we should change this line to "A JKube Generator is composed of opinionated image configurations for building container images, this image configuration is later used by enrichers to generate Kubernetes resources"

Reply:
OK Somehow I miss understood the scope of generator, I will provide another proposal for this part